napari/layers/_image_layer/view/properties.py

```diff
@@ -13,12 +13,14 @@ def __init__(self, layer):
         comboBox = QComboBox()
         for cmap in self.layer.colormaps:
             comboBox.addItem(cmap)
+        comboBox._allitems = set(self.layer.colormaps)
         index = comboBox.findText('magma', Qt.MatchFixedString)
```

Comment:
This should be 
`index = comboBox.findText(self.layer.colormap_name, Qt.MatchFixedString)`
so that if the default colormap ever changes in the model file it will change here automatically

Reply:
Got it, thanks, and thanks for fixing! =)